src/lib/list/list.scss

```diff
@@ -2,6 +2,7 @@
 @import '../core/style/list-common';
 @import '../core/style/layout-common';
 @import '../divider/divider-offset';
+@import '../../cdk/a11y/a11y';
```

Comment:
Can you please add the a11y dependency to the bazel build file? 

It currently just depends on the `a11y`  `sass_library` transitively being available in the Bazel deps

```bzl
sass_binary(
  name = "list_scss",
  src = "list.scss",
  deps = [
    "//src/cdk/a11y:a11y_scss_lib",  # <---
    "//src/lib/core:core_scss_lib",
    "//src/lib/divider:divider_scss_lib"
  ],
)
```

I already walked through all build files and enforced this manually, but there is no real enforcement check yet.